shuup_tests/default_tax/test_default_tax.py

```diff
@@ -182,6 +182,23 @@ def test_rule_min_max():
     postal_code = None
     assert TaxRule.objects.may_match_postal_code(postal_code).count() == 1
 
+@pytest.mark.django_db
+def test_wildcard_postalcode():
+    tax = create_tax("test-1", rate=Decimal("0.12"))
+    tax.save()
+    rule = TaxRule.objects.create(postal_codes_pattern="*", tax=tax)
+    assert rule._postal_codes_min == "*"
+    assert rule._postal_codes_max == "*"
```

Comment:
IMHO these asserts are wrong. You shouldn't assert about the inner state of the object. What matters is that it matches correctly (the assert below), it doesn't matter how (the asserts here)...